### PR TITLE
King feature

### PR DESCRIPTION
**Pull request by etechstaff**

### CODE REVIEW

**Comment by etechstaff** [comment]
everything looks good

File: file2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+list of ingeneers
```
Comment (ItodoI) [inline file2:1]: Looks good..

**Review by ItodoI — COMMENTED** [review]
I approve

File: file2 [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+king
```
Comment (ItodoI) [inline file2:2]: two team 4 memebers still out

**Pull request merged**